openpype/hosts/webpublisher/webserver_service/webpublish_routes.py

```diff
@@ -359,12 +359,19 @@ async def get(self, project_name=None) -> Response:
             "studio_exts": set(["psd", "psb", "tvpp", "tvp"])
         }
         collect_conf = sett["webpublisher"]["publish"]["CollectPublishedFiles"]
-        for _, mapping in collect_conf.get("task_type_to_family", {}).items():
-            for _family, config in mapping.items():
-                if config["is_sequence"]:
-                    configured["sequence_exts"].update(config["extensions"])
-                else:
-                    configured["file_exts"].update(config["extensions"])
+        configs = collect_conf.get("task_type_to_family", [])
```

Comment:
Few questions:
1. Why are you loading settings manyally and not using `self.task_type_to_family`?
2. Why is the task type in settings if is not used?
3. Shouldn't you store the family from key of `conf_mappings` if `conf_mappings` is dictionary?

Reply:
Nevermind. I though this is in the collector but this is differnt file. Damn...

Reply:
Yes, this is only for frontend to collect which formats are "allowed" eg. configured. Currently only per project_name.